torch/testing/_asserts.py

```diff
@@ -796,6 +633,24 @@ def assert_close(
 
     For ``max_abs_diff`` and ``max_rel_diff`` the type depends on the :attr:`~torch.Tensor.dtype` of the inputs.
 
+    .. note::
+
+        :func:`~torch.testing.assert_close` is highly configurable with strict default settings. Users are encouraged
+        to :func:`~functools.partial` it to fit their use case. For example, if you an equality check is needed one
+        might define an ``assert_equal`` that uses zero tolrances for every ``dtype`` by default:
+
+        .. code:: python
+
+        >>> import functools
+        >>> import torch
+        >>> assert_equal = functools.partial(torch.testing.assert_close, rtol=0, atol=0)
+        >>> assert_equal(1e-9, 1e-10)
+        AssertionError: Tensors are not close!
+
+        Mismatched elements: 1 / 1 (100.0%)
+        Greatest absolute difference: 8.999999703829253e-10 at 0 (up to 0 allowed)
+        Greatest relative difference: 8.999999583666371 at 0 (up to 0 allowed)
```

Comment:
This will make much more sense after #60091 from later in this stack is in. The error will read:

```
AssertionError: Scalars are not equal!

Absolute difference: 8.999999703829253e-10
Relative difference: 8.999999583666371
```